kite-data/kite-data-crunch/pom.xml

```diff
@@ -31,8 +31,20 @@
   </description>
 
   <build>
+    <resources>
+      <resource>
+        <directory>src/test/avro</directory>
```

Comment:
Okay, so kite-data-core does it so that the test schemas are included in the test-jar artifact. So it looks like this just includes it in the test jar correctly and not in the overall jar, so it is fine. Thanks!